Scripts/Editor/EnumFlagPropertyDrawer.cs

```diff
@@ -6,47 +6,51 @@
 using UnityEditor;
 using UnityEngine;
 
-[CustomPropertyDrawer(typeof(EnumFlagAttribute))]
-public class EnumFlagDrawer : PropertyDrawer
+namespace Fjord.Common.UnityEditor.PropertyDrawers
```

Comment:
All I did here was wrap this in a namespace, but the diff tool seems to think it's a radical content change.

Reply:
Looks like its because the indent formatting changed